src/app/shared/components/category-select/category-select.component.html

```diff
@@ -16,8 +16,8 @@
   <ng-container *ngIf="(departments$ | async)?.length">
     <label class="step-label">Відділ/профіль<span class="step-required">*</span></label>
     <input matInput type="text" placeholder="Почніть вводити відділ або виберіть зі списку" class="step-input"
-      formControlName="departmentId" [matAutocomplete]="autoDepartment" disableOptionCentering
-      panelClass="dropdown-panel" (click)="getFullDepartmentList()" autocomplete="off">
+      formControlName="departmentId" [matAutocomplete]="autoDepartment" disableOptionCentering (click)="getFullDepartmentList()"
+      panelClass="dropdown-panel" (focusout)="onFocusDepartmentOut(filteredDepartments[0], $event.target.value)" autocomplete="off">
```

Comment:
Is exist styleguide for this? Do we have to put here $event.target.value or just $event and then get value into method onFocusDepartmentOut?